pom.xml

```diff
@@ -483,7 +483,7 @@
     </distributionManagement>
 
     <properties>
-        <carbon.identity.version>5.25.204</carbon.identity.version>
+        <carbon.identity.version>5.25.286</carbon.identity.version>
```

Comment:
Need to bump the version again after merging https://github.com/wso2/carbon-identity-framework/pull/4860#pullrequestreview-1582142694

Reply:
yes, will bump

Reply:
Resolved by https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/162/commits/935ea6c0fdab0e5892f6c51319144c8847cbbba7